v03_pipeline/bin/dataproc_vep_init_38.sh

```diff
@@ -0,0 +1,58 @@
+#
+# VEP init action for dataproc
+#
+# adapted/copied from
+# https://github.com/broadinstitute/gnomad_methods/blob/main/init_scripts/vep105-init.sh
+# and gs://hail-common/hailctl/dataproc/0.2.128/vep-GRCh38.sh
+#
+
+set -x
+
+export PROJECT="$(gcloud config get-value project)"
+export ENVIRONMENT="$(/usr/share/google/get_metadata_value attributes/ENVIRONMENT)"
+export VEP_CONFIG_PATH="$(/usr/share/google/get_metadata_value attributes/VEP_CONFIG_PATH)"
+
+# Install docker
+apt-get update
+apt-get -y install \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    gnupg2 \
+    software-properties-common \
+    tabix
+curl -fsSL https://download.docker.com/linux/debian/gpg | sudo apt-key add -
+sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/debian $(lsb_release -cs) stable"
+sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/debian $(lsb_release -cs) stable"
+apt-get update
+apt-get install -y --allow-unauthenticated docker-ce
+
+# https://github.com/hail-is/hail/issues/12936
+sleep 60
+sudo service docker restart
+
+gcloud storage cp gs://seqr-luigi/releases/$ENVIRONMENT/latest/var/vep_config/vep-GRCh38.json $VEP_CONFIG_PATH
+
+cat >/vep.c <<EOF
+#include <unistd.h>
+#include <stdio.h>
+
+int
+main(int argc, char *const argv[]) {
+  if (setuid(geteuid()))
+    perror( "setuid" );
+
+  execv("/vep.bash", argv);
+  return 0;
+}
+EOF
+gcc -Wall -Werror -O2 /vep.c -o /vep
+chmod u+s /vep
+
+gcloud storage cp gs://seqr-luigi/releases/$ENVIRONMENT/latest/bin/download_vep_data.bash /download_vep_data.bash
```

Comment:
I refactored this so that shared code with the local deploy would be modularized (in bash!! 😬).  I'm endeavoring to test on dataproc right now.

Reply:
No, it isn't intended for local users.  It's an "initialization action" for dataproc that hail has typically maintained, but we deviated from with our VEP version bump.  `git` isn't doing a great job with the diff because I [renamed the file](https://github.com/broadinstitute/seqr-loading-pipelines/pull/841/files/dd87fef8cb96ac9d1a6874566ac795c193f58f00#diff-6f8303e2a8204e628c6f8d80ddd6d856e1a104d06807cc82c0b4430383783456L1)